### DISCUSSION TITLE
[Feature Request] Create HeaderGroup component

### DISCUSSION THREAD

**Discussion by evandrmb** [heroui-inc/heroui#4733] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently, there's no way of using `Table` with a Header Group or even multiple Header's

### Describe the solution you'd like

I expect to be able to create grouped headers with N columns and each group should have their data, something like Accordion but for tables

### Describe alternatives you've considered

I believe the best approach is to create a component called `HeaderGroup` with `columns` props.

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1399/[feature-request]-create-headergroup-component">ENG-1399 [Feature Request] Create HeaderGroup component</a></p>
